linode/manager
# refactor: [M3-6522] - Toggle component v7 story migration and cleanup

## Description 📝
Small PR to migrate Toggle component story to a V7 (.tsx) story!

## Changes  🔄
List any change relevant to the reviewer.
- New v7 .tsx story to replace the .mdx one
- Import/export cleanup + eslint fixes on files I visited

## How to test 🧪

### Prerequisites
- pull code locally
- run yarn && yarn storybook

### Verification steps 
- navigate to http://localhost:6006/?path=/docs/components-toggle--documentation + confirm story loads properly
- Confirm no regression for toggle on CM

## As an Author I have considered 🤔

*Check all that apply*
